assets/src/edit-story/components/helpCenter/useHelpCenter/index.js

```diff
@@ -177,7 +179,8 @@ export function useHelpCenter() {
             web_stories_onboarding: createBooleanMapFromKey(persistenceKey),
           },
         }).catch(actions.persistingReadTipsError),
-      TRANSITION_DURATION
+      // duration of menu transition which takes the longest
+      TRANSITION_DURATION * 1.2
```

Comment:
should we update `TRANSITION_DURATION` directly?

Reply:
We could, but we'd then have to go update all the transitions that consume it and alter accordingly. Since this is a short term solution and gonna get ripped out soon anyway, I'm fine with leaving it as is for now.